frameworks/Swift/vapor/Sources/vapor-tfb/config.swift

```diff
@@ -0,0 +1,28 @@
+import Vapor
+import TfbCommon
+
+/// Called before your application initializes.
+///
+/// [Learn More →](https://docs.vapor.codes/3.0/getting-started/structure/#configureswift)
+public func configure(
+    _ config: inout Config,
+    _ env: inout Environment,
+    _ services: inout Services
+) throws {
+
+    services.register(EngineServerConfig.default(hostname: "0.0.0.0"))
+    services.register(ServerMiddleware.self)
+
+    // Register routes to the router
+    let router = EngineRouter.default()
+    try routes(router)
+    services.register(router, as: Router.self)
+
+    // Register middleware
+    var middlewares = MiddlewareConfig()
+    middlewares.use(ServerMiddleware.self)
+    middlewares.use(DateMiddleware.self)
```

Comment:
this can actually be removed now. HTTP 3.0.4 (just released an hour ago) automatically serializes the `Date` header. in a much more performant way, too.